umap/models.py

```diff
@@ -193,6 +194,40 @@ class Map(NamedModel):
     objects = models.Manager()
     public = PublicManager()
 
+    @property
+    def unique_id(self):
+        return f"map_{self.pk}"
```

Comment:
This sounds like more related to HTML than models, shouldn't we keep `map_` inside the html and just use `pk` ?